Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerEventsTests.cs

```diff
@@ -124,27 +133,45 @@ private PointerHandler SetupScene()
             var collider = testContent.GetComponentInChildren<Collider>();
             Assert.IsNotNull(collider);
 
-            return collider.gameObject.AddComponent<PointerHandler>();
+            handler = collider.gameObject.AddComponent<PointerHandler>();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            TestUtilities.ShutdownMixedRealityToolkit();
         }
 
-        // Tests that the right pointer events are raised when using a single pointer to interact with a collider
         [UnityTest]
-        public IEnumerator TestOnePointer()
+        public IEnumerator Test()
         {
-            var handler = SetupScene();
-            var rightHand = new TestHand(Handedness.Right);
+            // Step once to receive focus events from gaze pointer
+            yield return null;
+
+            var expected = new PointerHandler.State();
+            expected.numBeforeFocusChange++;
```

Comment:
Use explicit numbers instead of increment to make clear what we are expecting here.

Reply:
I think it is not too confusing to rely on the state being initialized to zero.